recipes/temporalio-cli/recipe.yaml

```diff
@@ -0,0 +1,54 @@
+context:
+  version: "1.3.0"
+
+package:
+  name: temporalio-cli
+  version: ${{ version }}
+
+source:
+  url: https://github.com/temporalio/cli/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 15be9f155cd5114367942568f884969f7ed2d3262ad39bb665cf359735f643b3
+  target_directory: src
+
+build:
+  number: 0
+  script:
+    - cd src
+    - go-licenses save ./cmd/temporal --save_path ../library_licenses --ignore github.com/cactus/go-statsd-client/statsd --ignore modernc.org/mathutil
```

Comment:
could you manually add the missing license files into the feedstock (or add them as an additional source)?

Reply:
Done!